tests/threshold_tools/test_thresholds_returns.py

```diff
@@ -42,6 +42,16 @@ def test_return_value(T2_ams: xr.DataArray):
     assert not np.isnan(rvs["return_value"].values[()])
 
 
+@pytest.mark.advanced
+def test_return_value_dropna(T2_ams: xr.DataArray):
+    """Test Return Values with dropna_time option."""
+    T2_ams.data[2] = np.nan
+    rvs = threshold_tools.get_return_value(
+        T2_ams, return_period=10, distr="gev", bootstrap_runs=1, multiple_points=False
```

Comment:
This test doesn't have `dropna_time=True` passed in?

Reply:
Well then. I'll fix that